chapter07/rwd/ex4-7.pl

```diff
@@ -0,0 +1,15 @@
+#!/urs/bin/perl
+
+use strict;
+use warnings;
+
+if (@ARGV != 1) {
+	die "Error: file name must be supplied.";
+}
+
+while (<>) {
+	if (/[A-Z][a-z]+/) {
```

Comment:
This will only work for words like This Where Only The First Letter Is Capitalized, and not funny words like iPod